packages/react-components/react-nav-preview/stories/src/Nav/index.stories.tsx

```diff
@@ -5,7 +5,7 @@ import { Nav } from '@fluentui/react-nav-preview';
 // import bestPracticesMd from './NavBestPractices.md';
 
 export { Basic } from '../NavDrawer/Basic.stories';
-export { VariableSizedItems } from '../NavDrawer/VariableSizedItems.stories';
+export { VariableDensityItems } from '../NavDrawer/VariableDensityItems.stories';
```

Comment:
Story is missing (removed instead of renamed?)

Reply:
awesome catch - thank you!

Reply:
fixed